atom/browser/native_window_mac.mm

```diff
@@ -740,8 +740,9 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
 
 void NativeWindowMac::Show() {
   if (is_modal() && parent()) {
-    [parent()->GetNativeWindow() beginSheet:window_
-                          completionHandler:^(NSModalResponse) {}];
+    if ([window_ sheetParent] == nil)
```

Comment:
@zcbenz I think we want to return early even when `Show` is called on a window with a sheet parent so that is why I did the `if` check under the `is_modal() && parent()` check.